app/admin/models/Menu_options_model.php

```diff
@@ -53,7 +53,7 @@ class Menu_options_model extends Model
         ['locations.*', 'lang:admin::lang.label_location', 'integer'],
     ];
 
-    protected $purgeable = ['option_values'];
+    protected $purgeable = ['option_values', 'overwrite_existing_items'];
```

Comment:
Done.
New column so that it persists?

Reply:
Yup

Reply:
Say in future we have some background process checking these things, it can easily determine what action to take by checking the column value. (the best explanation I can think of ;))

Reply:
Sounds good, added a migration and made it persist